README.md

```diff
@@ -556,17 +556,17 @@ The protocols enabled for the site. If 'https' is specified, 'http' is implied.
 
 ##### `bindings`
 
-The protocol, address, port, and ssl certificate bindings for a web site.
+The protocol, address, port, and ssl certificate bindings for a website.
 
-The bindinginformation value should be in the form of the IPv4/IPv6 address or wildcard `*`, then the port, then the optional hostname separated by colons:  `(ip|\*):[1-65535]:(hostname)?`
+The binding information value should be in the form of the IPv4/IPv6 address or wildcard `*`, then the port, then the optional hostname separated by colons:  `(ip|\*):[1-65535]:(hostname)?`
```

Comment:
There actually is a property in the hash you pass to this property called `bindinginformation` so that's probably what "binding information" should be switched to. \`bindinginformation\` with the back ticks so it comes through properly in the rendered markdown.